src/main/java/org/jboss/threads/Version.java

```diff
@@ -31,6 +31,9 @@ public final class Version {
     private Version() {
     }
 
+    // No specific prefix here, this controls the behavior of multiple JBoss libraries
+    private static final String JBOSS_LOG_VERSIONS = "jboss.log-versions";
```

Comment:
Wondering if we should make this public for convenience but I'm not sure we want to commit to it always being around.

Reply:
This is more of a convention so I'd opt for it not being public. More than likely too you'd want the property configured before you access the type as logging would need to be configured first.

Reply:
Ah ah yeah. I thought of that yesterday and then forgot today why I didn’t post the proposal in the end :).